bedrock/firefox/templates/firefox/firefox-quantum.html

```diff
@@ -0,0 +1,170 @@
+          <p>
+          {% trans %}
+            Hello, gorgeous! Firefox’s sleek, new look comes loaded with intuitive features
+            like in-browser screenshots and more.
+          {% endtrans %}
+          </p>
+        </div>
+      </section>
+    </div>
+  </div>
+  <section class="other-features" data-scroll-tracking="Other Features">
+    <div class="content">
+      <h2>{{ _('Get the job done… faster') }}</h2>
+      <ul>
+        <li>
+          <h3 class="new-tab">{{ _('New Tab') }}</h3>
+          <p>{{ _('Search across multiple sites, view your top pages and discover new content.') }}</p>
+        </li>
+        <li>
+          <h3 class="library">{{ _('Library') }}</h3>
+          <p>{{ _('Enjoy everything you’ve saved to Firefox while browsing in one, easy place.') }}</p>
+        </li>
+        <li>
+          <h3 class="extensions">{{ _('Extensions') }}</h3>
+          <p>{{ _('Personalize Firefox with your favorite extras that help you do you.') }}</p>
+        </li>
+        <li>
+          <h3 class="mobile-browsing">{{ _('Mobile Browsing') }}</h3>
+          <p>{{ _('Access your bookmarks, open tabs and passwords across all your devices.') }}</p>
+        </li>
+      </ul>
+    </div>
+  </section>
+  <div class="key-features-section" data-scroll-tracking="Second Key Features">
+    <div class="content">
+      <section class="key-feature privacy">
+        <div class="key-feature-content">
+          <h2>{{ _('Powerful privacy') }}</h2>
+          <p>
+          {% trans url='https://blog.mozilla.org/firefox-de/die-studie-zum-schutz-vor-aktivitatsverfolgung/' if LANG == 'de' else 'https://blog.mozilla.org/firefox/tracking-protection-study/' %}
+            You’re in control of your online information. Use Firefox Private Browsing to block ads
+            with trackers for extra peace of mind… and pages that load <a href="{{ url }}">up to 44% faster</a>.
+          {% endtrans %}
+          </p>
+        </div>
+      </section>
+      <section class="key-feature mozilla">
+        <div class="key-feature-content">
+          <h2>{{ _('Browse for good') }}</h2>
+          <p>
+          {% trans %}
+            Firefox is backed by the non-profit Mozilla, who keeps the Internet healthier through programs
+            that support tech education for girls, create trust around factual news, bring civility to the
+            comments section and more.
+          {% endtrans %}
+          </p>
+        </div>
+      </section>
+    </div>
+  </div>
+  <section class="secondary-download-cta" data-scroll-tracking="Secondary Download CTA">
+    <div class="content">
+      <div class="secondary-download-cta-content">
+        <h2>{{ _('Start browsing with <strong>Firefox</strong> Quantum') }}</h2>
+        {{ download_firefox(alt_copy=_('Download Now'), dom_id='firefox-desktop-page', download_location='other', button_color='quantum-download') }}
```

Comment:
This is a duplicate ID: `dom_id='firefox-desktop-page'` - it should be different to the one in the primary cta section at the top of the page

Reply:
Doh!